packages/experience-builder-sdk/src/communication/MouseOverHandler.ts

```diff
@@ -5,24 +5,37 @@ import { sendMessage } from './sendMessage';
 export class MouseOverHandler {
   private currentHoveredElementId: string | null = null;
 
+  private getMargins = (element: HTMLElement) => {
+    if (!window) return undefined;
```

Comment:
```suggestion
    if (typeof window !== 'undefined') return undefined;
```

I may be wrong, but I think the current check would still cause an error during SSR.

Reply:
Oh, I did not know this, thanks 👍